controllers/handlers/kubevirt_test.go

```diff
@@ -1964,6 +1975,51 @@ Version: 1.2.3`)
 					Expect(existingResource.Annotations).ToNot(HaveKey(kubevirtcorev1.EmulatorThreadCompleteToEvenParity))
 				})
 
+				It("should add the Passt Binding to Kubevirt CR if PasstNetworkBinding is true in HyperConverged CR", func() {
+					hco.Annotations[deployPasstNetworkBindingAnnotation] = "true"
+					hco.Spec.NetworkBinding = nil
+
+					existingResource, err := NewKubeVirt(hco)
```

Comment:
I know, there are tons of `existingResource` vars all over the tests. But these are leftovers. Please call it `kv` to be more clear about what it is (+ few more instances below).